src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,15 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
+
+        Ball[] res = new Ball[] {ball1, ball2, ball3};
```

Comment:
Instead of manually creating an array with the balls, you could use a loop to fill the array. This would make your code more scalable and maintainable if the number of balls were to change.

Reply:
not fixed
call getrandomball in a loop, don't create temporary local variables, don't create array